engine/src/test/java/org/destinationsol/testsupport/TestGameStateObject.java

```diff
@@ -0,0 +1,38 @@
+package org.destinationsol.testsupport;
+
+import com.badlogic.gdx.Application;
+import org.destinationsol.modules.ModuleManager;
+import org.terasology.gestalt.entitysystem.component.management.ComponentManager;
+
```

Comment:
I'm not clear exactly what this class does. I can tell it's some sort of data structure for the game state, but I don't know exactly what it does or what the fields represent. A little bit of Javadoc would go a long way ;)